File: prolog-3-week/dayThree.pl
game() :-
    writeln('choose number between 1 and 10'),
    random(1, 10, RndVal),
    read(Number),
    game_loop(Number, RndVal).

game_loop(Number, RndVal) :-
    dif(Number, RndVal),
    (Number < RndVal ->
        writeln('Your number is lower than the answer');
        writeln('Your number is greater than the answer')
    ),
    read(NewNumber),
    game_loop(NewNumber, RndVal).

game_loop(Number, RndVal) :-
    Number = RndVal,
    writeln('You won!').

File: scala-4-week/DayThree.scala
object Game {
  def startGame() = {
    val rand = new scala.util.Random
    val rndValue = rand.between(1, 10)

    println("choose number between 1 and 10")
    var input = scala.io.StdIn.readLine().toInt
    while(input != rndValue) {
      if(input < rndValue) {
        println("Your number lower than answer")
      } else {
        println("Your number greater than answer")
      }
      input = scala.io.StdIn.readLine().toInt
    }
    println("You won!")
  }

  def main(args: Array[String]): Unit = {
    startGame()
  }
}